### DISCUSSION TITLE
How prevent docusaurus from generating page in `/docs/tags`

### DISCUSSION THREAD

**Discussion by KyrietS** [facebook/docusaurus#5542] [Q&A] [answered]
Hi, I've just upgraded docusaurus to the latest version and I see a warning message:

> Attempting to create page at /docs/tags, but a page already exists at this route

Docusaurus tries to generate some kind of page with tags and reserves `/docs/tags` for this. I don't want this because we have a doc page under `/docs/tags` and it's important to leave this URL as it is (I don't want to use slug). 

Is there a way to change the default path where Docusaurus generates tags? Or is it possible to disable tags completely for docs?

This PR introduced tags for docs: https://github.com/facebook/docusaurus/pull/3646 but I can't see any config to disable this feature.

**Comment by Josh-Cena** [comment, marked as answer]
Currently the `/docs/tags` route is hardcoded and there's no way to disable tags feature either: https://github.com/facebook/docusaurus/blob/02eee61aa540b8e839e8a7e17c6910e411c5d95b/packages/docusaurus-plugin-content-docs/src/versions.ts#L401-L403

There seems to be a plan to make the path configurable, and you probably would give this a pretty solid use-case :P

**Reply by KyrietS** [reply]
Thanks!

**Reply by lex111** [reply]
Definitely route path to tags page should be configurable, by analogy with archive page. This will be implemented in PR #5545.
